sites/public/pages/finder.tsx

```diff
@@ -197,7 +230,7 @@ const Finder = () => {
                 </div>
 
                 <div className="flex justify-center align-center bg-white py-8">
-                  <a className="underline" href="/listings">
+                  <a className="underline" onClick={skipToListings}>
```

Comment:
We can't have an anchor tag without an href for a11y - we should use an anchor + href or a button + onClick